app/main.py

```diff
@@ -1,3 +1,69 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    # +
+    def __add__(self, other: Distance | int | float) -> Distance:
+        if isinstance(other, Distance):
+            return Distance(km=self.km + other.km)
+
+        return Distance(km=self.km + other)
+
+    # +=
+    def __iadd__(self, other: Distance | int | float) -> Distance:
+        if isinstance(other, (int, float)):
+            self.km += other
+            return self
+
+        self.km += other.km
+        return self
+
+    # *
+    def __mul__(self, other: int) -> Distance:
+        return Distance(km=self.km * other)
+
+    # /
+    def __truediv__(self, other: int | float) -> Distance:
+        if other == 0:
+            return Distance(other)
```

Comment:
The `__truediv__` method should raise an exception when dividing by zero instead of returning a `Distance` object with zero. Consider using `raise ZeroDivisionError('division by zero')`.